subsquid/src/pabloProcessor.ts

```diff
@@ -118,8 +122,8 @@ export async function processPoolCreatedEvent(ctx: EventHandlerContext, event: P
     }
 }
 
-export function createPoolAssetId(poolId: string, assetId: bigint): string {
-    return poolId + '-' + assetId;
+export function createPoolAssetId(eventId: string, poolId: bigint, assetId: bigint): string {
+    return eventId + '-' + poolId + '-' + assetId;
```

Comment:
@vimukthi-git how does adding an event id solve the problem of querying initial balances within the pool?

Reply:
so is this eventId a timestamp?

Reply:
No, it is just a unique id, that allows to store all historic versions of an entity without conflict. Timestamps already exists in the entities.

Reply:
may be 
```
sha256(`${eventId}-${poolId}-${assetId}`)
```
better?